generators/client/templates/angular/src/main/webapp/app/home/home.css.ejs

```diff
@@ -30,9 +30,8 @@ Main page styles
 
 /* wait autoprefixer update to allow simple generation of high pixel density media query */
 @media only screen and (-webkit-min-device-pixel-ratio: 2),
-    only screen and (min--moz-device-pixel-ratio: 2),
+    only screen and (-moz-min-device-pixel-ratio: 2),
```

Comment:
update this for react and sass files as well

Reply:
@deepu105 The react version of this file is quite different and doesn't contain the issue. Should it be synched to match?

Reply:
Fixed for sass. React file doesn't have the same layout. If needed to sync let me know and I will add to the PR for this and the other files where they vary.

Reply:
the structure is different please search for occurrence of these in the codebase, and no need to sync

Reply:
Done. `min-device-pixel-ratio` only occurs in the two updated files for angular and sass and both are patched.